### DISCUSSION TITLE
Simulate "Not Published" Status in Preview Environment

### DISCUSSION THREAD

**Discussion by luispozoWS** [OP-TED/eForms-SDK#973] [Q&A] [answered]
Helllo,

Is it possible for you to reject our form 34471-2024 - SDK 1.9 in order to simulate "Not Published" in Preview Environnment ?

Best regards

[https://github.com/OP-TED/eForms-SDK/discussions/897](https://urldefense.com/v3/__https:/github.com/OP-TED/eForms-SDK/discussions/897__;!!NEMsmePo_HYI!d9S5gwop9RoAWXoKDMw7hxPBxz1yb92hwARY_iS5c_U8z8tBIJoFQrJ2tUMUix72jxT-vFeASPCou2m64vBNKmOFkbDpAVzv$)

**Comment by YvesJo** [comment, marked as answer]
Hi,
For such questions, please get in touch directly with the helpdesk and provide the Notice ID as well as the version ID.
KR,
